cmd/pitr/collector/collector.go

```diff
@@ -219,7 +219,7 @@ func (c *Collector) CollectBinLogs() error {
 		}
 
 		if lastUploadedBinlogName == "" {
-			log.Println("Gap detected in the binary logs. Binary logs will be uploaded anyway, but full backup needed for consistent recovery.")
+			return errors.New("Gap detected in the binary logs. Binary logs will be uploaded anyway, but full backup needed for consistent recovery.")
```

Comment:
@hors I'm not sure if we should return or not? We can also just put `ERROR` prefix to this log

Reply:
OK, returning an error here will stop execution and binlogs won't be uploaded and this contradicts with the message. I'll just put `ERROR` prefix